api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -1,5 +1,7 @@
 package org.pmiops.workbench.firecloud;
 
+import static org.pmiops.workbench.api.ClusterController.WORKSPACE_DELIMITER;
```

Comment:
Could you put this constant somewhere central? This creates a bit of an awkward dependency graph (low level service referencing a controller).